components/scream/src/control/atmosphere_driver.cpp

```diff
@@ -215,20 +217,28 @@ void AtmosphereDriver::create_fields()
   m_ad_status |= s_fields_created;
 }
 
-void AtmosphereDriver::initialize_output_manager () {
+void AtmosphereDriver::initialize_output_managers (const bool restarted_run) {
   check_ad_status (s_comm_set | s_params_set | s_grids_created | s_fields_created);
 
-  // Create Initial conditions
-
-  // Create Output manager
-  if (m_atm_params.isSublist("Output Manager")) {
-    auto& out_params = m_atm_params.sublist("Output Manager");
-    m_output_manager.set_params(out_params);
-    m_output_manager.set_comm(m_atm_comm);
-    m_output_manager.set_grids(m_grids_manager);
-    m_output_manager.set_field_mgr(get_ref_grid_field_mgr());
+  // For each grid in the grids manager, grab a homonymous sublist of the
+  // "Output Managers" atm params sublist (might be empty), and create
+  // an output manager for that grid.
+  // in the parameter list with that name. If so, create an output
```

Comment:
This comment reads as though something is missing.

Reply:
Ah, yes, I change my mind when writing it, and left chunks of the first draft around. Will clean up.